articleEditor/editor.js

```diff
@@ -19,21 +17,28 @@
     CarouselMaket(window.articlesFromLS);
     reattachListeners();
 
-    ["closeContent", "addContent"].map(x => document.querySelector(`.${x}`).addEventListener(`click`, () => {
-        inputValues();
-        ["show"].map(v => window.page_content_wrapper.classList.toggle(v));
-        setTimeout(() => {
-            document.querySelector("footer").scrollIntoView({behavior: "smooth", block: "end", inline: "nearest"});
-        },10);
-        window.actionForm.setAttribute(`data-action`, `save`);
+    ["closeContent", "addContent"].map(x => document.querySelector(`.${x}`)
+        .addEventListener(`click`, () => {
+            inputValues();
+            ["show"].map(v => window.page_content_wrapper.classList.toggle(v));
+            setTimeout(() => {
+                document.querySelector("footer").scrollIntoView({
+                    behavior: "smooth", block: "end",
+                    inline: "nearest"
+                });
```

Comment:
Exclude it to a function `scrollTo($node)` as you use it more than 2 times in project.